website/src/app/[lang]/[region]/(website)/me/layout-client.tsx

```
@@ -149,6 +154,11 @@ export function LayoutClient({ params, translations, children }: PropsWithChildr
 					</div>
 				</CollapsibleTrigger>
 				<CollapsibleContent className="-mt-10 mb-12 border-b md:hidden">{isOpen && navigationMenu}</CollapsibleContent>
+				{auth.currentUser?.isAnonymous && (
+					<Card className="mb-12">
+						<Typography className="m-4" dangerouslySetInnerHTML={{ __html: translations.emailLoginRequired }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)